src/background/sidebar-injector.js

```diff
@@ -239,6 +251,10 @@ export function SidebarInjector() {
       await injectIntoPDF(tab);
     } else if (type === CONTENT_TYPE_VITALSOURCE) {
       await injectIntoVitalSourceReader(tab, config);
+    } else if (type === CONTENT_TYPE_LMS) {
+      throw new BlockedSiteError(
```

Comment:
Can you add a comment here to indicate why this site is blocked, as it might not be obvious to future developers.